b2/bucket.py

```diff
@@ -393,9 +400,55 @@ def _upload_large_file(
         response = self.api.session.finish_large_file(file_id, part_sha1_array)
         return FileVersionInfoFactory.from_api_response(response)
 
+    def _find_unfinshed_file(self, upload_source, file_name, file_info, part_ranges):
+        for file in self.list_unfinished_large_files():
+            if file.file_name == file_name and file.file_info == file_info:
+                files_match = True
+                finished_parts = []
+                for part in self.list_parts(file.file_id):
+                    # Compare part sizes
+                    (offset, part_length) = part_ranges[part.part_number - 1]
```

Comment:
unnecessary parenthesis (they make yapf slower and the code less readable (for me))